src/main/java/com/palantir/stash/stashbot/jobtemplate/JenkinsJobXmlFormatter.java

```diff
@@ -68,7 +68,7 @@ private String curlCommandBuilder(Repository repo, JobTemplate jobTemplate,
         final JenkinsServerConfiguration jsc = cpm
             .getJenkinsServerConfiguration(rc.getJenkinsServerName());
         StringBuffer sb = new StringBuffer();
-        sb.append("/usr/bin/curl -s -i ");
+        sb.append("/usr/bin/curl -k -s -i ");
```

Comment:
This really shouldn't be necessary...  but I can't think of any serious problems with a MITM attack since no creds are used here.